documentation/modules/ROOT/pages/configuration/filtering.adoc

```diff
@@ -96,6 +96,10 @@ value.op == 'u' && value.before.id == 2
 |Default
 |Description
 
+|`topic.regex`
```

Comment:
Should we consider adding the configuration property links like we've began doing for the connectors?